lisa/tools/hyperv.py

```diff
@@ -468,13 +472,13 @@ def enable_internal_dhcp(self, dhcp_scope_name: str = "DHCPInternalNAT") -> None
 
         # Configure the DHCP server to use the internal NAT network
         powershell.run_cmdlet(
-            f'Add-DhcpServerV4Scope -Name "{dhcp_scope_name}" -StartRange 192.168.5.50 -EndRange 192.168.5.100 -SubnetMask 255.255.255.0',  # noqa: E501
+            f'Add-DhcpServerV4Scope -Name "{dhcp_scope_name}" -StartRange {self.internal_nat_dhcp_ip_start} -EndRange {self.internal_nat_dhcp_ip_end} -SubnetMask 255.255.255.0',  # noqa: E501
```

Comment:
Remove `# noqa: E501`. Only url needs it for copying/opening easily. Otherwise, break the string can improve readability.

Reply:
Removed # noqa: E501 from one more place along with this one.